### PR TITLE
feat: logging message for tb upload completion.

### PR DESCRIPTION
**Pull request by ioga**
## Description

`Waiting for Tensorboard files to finish uploading` logging message is the last before shutdown. When something is wrong in shutdown, it makes it [confusing](https://hpe-aiatscale.slack.com/archives/C04C9JXB1C2/p1682962118013919?thread_ts=1682961986.088279&cid=C04C9JXB1C2) whether tensorboard is to blame. Let's add a completion message.

## Test Plan

N/A

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by stoksc — APPROVED** [review]

**Comment by stoksc** [comment]
(not an mlsys approver, but this looks good to me)

((possibly the logging is too verbose altogether, though, idk mlsys's logging strategy))

File: harness/determined/tensorboard/base.py [hunk 1]
```diff
@@ -174,3 +174,4 @@ def close(self) -> None:
         while self.is_alive():
             logging.info("Waiting for Tensorboard files to finish uploading")
             self.join(10)
+        logging.info("Tensorboard upload completed")
```
Comment (rb-determined-ai) [inline harness/determined/tensorboard/base.py:177]: I approve this pr.

But, while you are touching it, can you shuffle this code around so it doesn't basically always wait 10 seconds?  The way it works now there's a race condition that it almost always loses, even if there's no upload happening.

**Review by rb-determined-ai — APPROVED** [review]

File: harness/determined/tensorboard/base.py (hunk 1 above)
Comment (rb-determined-ai) [reply harness/determined/tensorboard/base.py:177]: disregard, this already has been fixed, I wasn't remembering correctly or thinking critically.

**Review by rb-determined-ai — APPROVED** [review]
even better

**Pull request merged**